### DISCUSSION TITLE
Add community sharing mechanism for additional prBodyNotes on packages

### DISCUSSION THREAD

**Discussion by gberche-orange** [renovatebot/renovate#24864] [Suggest an Idea]
### What would you like Renovate to be able to do?

as a renovate user
in order to discover additional important info related to a set of package upgrades
I need renovate to support and encourage the community to share with me this additional metadata such as related blogs, announcement slack, twitter acounts, etc...

### If you have any ideas on how this should be implemented, please tell us here.

Currently, I'm keeping private prBodyNotes such as 

```json

{
      "matchPackageNames": ["vmware-tanzu/carvel-kapp-controller"],
      "description": "blog + manual workaround for https://github.com/renovatebot/renovate/issues/17711",
      "prBodyNotes": ["Checkout Carvel blog at https://carvel.dev/blog/ "]
    },
   {
      "description": "tunes the PR body for concourse",
      "matchPackageNames": ["concourse"],
      "prBodyNotes": ["also checkout https://github.com/concourse/concourse/releases https://github.com/concourse/concourse/discussions"]
    },
    {
      "matchPackageNames": ["kyverno"],
      "customChangelogUrl": "https://github.com/kyverno/kyverno",
      "prBodyNotes": "Check out also https://kyverno.io/blog/"
    },    
    {
      "description": "crossplane main package is on the crossplane org. This should apply both to helm and any docker datasources",
      "matchPackageNames": ["crossplane"],
      "customChangelogUrl": "https://github.com/crossplane/crossplane/",
      "prBodyNotes": ["also checkout https://blog.upbound.io/, slack channel announcement https://crossplane.slack.com/archives/CEFQCGW1H, and [community meeting agenda](https://docs.google.com/document/d/1q_sp2jLQsDEOX7Yug6TPOv7Fwrys6EwcF5Itxjkno7Y)"]
    },
```

Somewhat relates to https://github.com/renovatebot/renovate/issues/18436

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
Could be added to https://github.com/renovatebot/renovate/blob/main/lib/modules/datasource/metadata-manual.ts

**Comment by gberche-orange** [comment]
would this be displayed *in addition* to the release notes ? 

also, it seems to me the current config file accepts a single entry per package, whereas an array is sometimes useful

https://github.com/renovatebot/renovate/blob/1127c5e4018f2fdd7883af5e5f59469c187e2a47/lib/modules/datasource/metadata-manual.ts#L3-L8

Can a given package be specified multiple times with different changelog urls ? The corresponding `customChangelogUrl` supports a single entry

https://docs.renovatebot.com/configuration-options/#customchangelogurl

**Comment by rarkins** [comment]
Packages should have a single source of truth for changelogs. What's an example of one where they split changelogs between two URLs and you need to read both?

**Comment by gberche-orange** [comment]
One example is crossplane where the changelog isn't sufficient to assess whether to merge the PR, and looking at community feedback on announcements, slack channel, and community meeting agenda is often useful:

`
 "customChangelogUrl": "https://github.com/crossplane/crossplane/",
      "prBodyNotes": ["also checkout https://blog.upbound.io/, slack channel announcement https://crossplane.slack.com/archives/CEFQCGW1H, and [community meeting agenda](https://docs.google.com/document/d/1q_sp2jLQsDEOX7Yug6TPOv7Fwrys6EwcF5Itxjkno7Y)"]
`

**Comment by rarkins** [comment]
In that case I think that having it as prBodyNotes makes most sense. It could generalized to a "additionalUrls" concept in the metadata, which is then auto generated into prBodyNotes
